src/oic/oauth2/__init__.py

```diff
@@ -649,6 +651,10 @@ def parse_response(
             except KeyError:
                 self.grant[_state] = self.grant_class(resp=resp)
 
+            if "id_token" in resp:
```

Comment:
This probably needs an additional check: `if self.sso_db is not None and "id_token" in resp:`, otherwise it can fail for OAuth?

Reply:
You're right.